### DISCUSSION TITLE
Docs website shows docs for v3 only

### DISCUSSION THREAD

**Discussion by mokhosh** [spatie/laravel-tags#502] [Q&A]
When I click on the [docs link](https://spatie.be/docs/laravel-tags/) it takes me to docs for v3, and there is no way to see docs for v4.

I realized this when I struggled for a few minutes with the publish commands not working:

```bash
php artisan vendor:publish --provider="Spatie\Tags\TagsServiceProvider" --tag="migrations"
```

This doesn't work because the actual tag in v4 is `tags-migrations`.

**Comment by inagamov** [comment]
@mokhosh thank you! for all people that struggle with it just use these commands instead of the ones from the v3 docs:

`php artisan vendor:publish --provider="Spatie\Tags\TagsServiceProvider" --tag="tags-migrations"`

`php artisan vendor:publish --provider="Spatie\Tags\TagsServiceProvider" --tag="tags-config"`

**Comment by inagamov** [comment]
@freekmurze would be nice to know if docs for v4 are different in any other place like migration and config. thanks
